src/dashboard/withdraw/PaymentMethods.tsx

```
@@ -0,0 +1,168 @@
+                color="secondary"
+                className="bg-dokan-btn hover:bg-dokan-btn-hover text-white"
+                onClick={ () => {
+                    window.location.href = bodyData?.data?.setup_url;
+                } }
+                label={ __( 'Setup', 'dokan-lite' ) }
+            />
+        );
+    };
+    if (
+        ! bodyData ||
+        ! bodyData.hasOwnProperty( 'isLoading' ) ||
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)